spring-integration-jdbc/src/main/resources/org/springframework/integration/jdbc/schema-sybase.sql

```diff
@@ -27,3 +27,11 @@ CREATE TABLE INT_MESSAGE_GROUP  (
 	UPDATED_DATE DATETIME DEFAULT NULL,
 	constraint MESSAGE_GROUP_PK primary key (GROUP_KEY, REGION)
 ) LOCK DATAROWS;
+
+CREATE TABLE INT_LOCK  (
```

Comment:
One objection: what would be the name of the new script (since the existing one is generic "schema", you can't create a new one without renaming the existing one, and that breaks existing code probably). I would prefer to just add the extra table.